qemu/tests/cfg/boot_cpu_model.cfg

```diff
@@ -49,9 +49,16 @@
                 - remove_flag:
                     cpu_model_GenuineIntel = "SandyBridge"
                     cpu_model_AuthenticAMD = "Opteron_G3"
-                    cpu_model_flags = ",-sse4a,-xsave,-aes,-sse4.2"
+                    cpu_model_flags = ",-sse4a,-xsave,-aes,-sse4.2,+ht"
+                    cpu_model_flags_AuthenticAMD = ",+cmp_legacy"
```

Comment:
Did you get the reason why there is cmp_legacy, but cmplegacy is definded in both host and guest